File: app/src/main/java/com/hana053/micropost/ui/Navigator.java
package com.hana053.micropost.ui;

import com.hana053.micropost.domain.User;

public interface Navigator {

    void navigateToUserShow(User user);

    void navigateToMain();

    void navigateToSignup();

    void navigateToLogin();

    void navigateToFollowings(long userId);

    void navigateToFollowers(long userId);

}

File: app/src/main/java/com/hana053/micropost/activity/Navigator.kt
package com.hana053.micropost.activity

interface Navigator {

    fun navigateToTop()
    fun navigateToMain()
    fun navigateToLogin()
    fun navigateToSignup()
    fun navigateToUserShow(userId: Long)
    fun navigateToFollowerList(userId: Long)
    fun navigateToFollowingList(userId: Long)
    fun navigateToMicropostNew()

}
